src/webgl/loading.js

```diff
@@ -38,11 +47,15 @@ require('./p5.Geometry');
  * </code>
  * </div>
  */
-p5.prototype.loadModel = function ( path ) {
+p5.prototype.loadModel = function ( path, normalized ) {
```

Comment:
I wonder whether we should use 'normalize', instead of 'normalized'.  My thinking is that since we want to normalize the model as an action, it is kinda happening in the present/future tense instead of 'normalized' which feels like the model has already been normalized.   What do you think?